### DISCUSSION TITLE
Terrain modeling in ERF

### DISCUSSION THREAD

**Discussion by indra098124** [AMReX-Codes/amrex#3254] [General]
I would like to explore ERF to study the effect of terrains in LES. May I ask how do I include terrain in ERF simulations? Is there any example I can look at?  How do I give latitude, longitude in an ERF simulation and how to get initial conditions from a global database (ERA5, NCEP etc). 

Thank you

**Comment by WeiqunZhang** [comment]
Maybe you could move the discussion to https://github.com/erf-model/ERF since it's specific ERF.

**Reply by indra098124** [reply]
That sounds great, however, there is no discussion section on [ERF github page](https://github.com/erf-model/ERF).

**Reply by WeiqunZhang** [reply]
You can post it an issue there.
